test/invoice-workflow.e2e-spec.ts

```
@@ -0,0 +1,884 @@
+
+      for (let i = 0; i < maxMonitoringAttempts; i++) {
+        await TestUtils.wait(200);
+
+        const tasksResponse = await request(app.getHttpServer())
+          .get(`/api/invoice/tasks/${customerId}`)
+          .expect(200);
+
+        const task = tasksResponse.body.tasks.find((t: any) => t.id === taskId);
+
+        if (task) {
+          console.log(`📊 Task status at attempt ${i + 1}: ${task.status}`);
```

ESLint

[error] Unexpected console statement.

(no-console)